iceoryx_posh/cmake/iceoryx_poshConfig.cmake

```diff
@@ -1,3 +1,17 @@
+# Copyright 2020 FIXME
```

Comment:
@ApexAI/team-i I'll need some help from you guys to figure out the correct author for some of these copyright headers. Thanks!

Reply:
@jpsamper2009  This should all be Bosch GmbH. I will ping some guys from Bosch who contribute to iceoryx so that they can fix that

Reply:
@elBoberido Thanks! Other than that this PR is ready

Reply:
@jpsamper2009 our workflow will be to have the feature branches in this fork and create PRs in https://github.com/eclipse/iceoryx
All the issues will also be created and tracked in upstream.
In order to merge the PR you need to sign the https://www.eclipse.org/legal/ECA.php. If you like to contribute frequently, which we would appreciate, I would recommend to sign it.
In the meantime, one of the iceoryx committer could contribute this changes to eclipse/iceoryx.

Reply:
@jpsamper2009 the copyright header were merged with https://github.com/eclipse/iceoryx/pull/339

do you know if we have to add the Apex copyright for such changes like adding the period the the source files?

Reply:
@elBoberido I don't think adding a period really warrants claiming any kind of ownership. However, I am not a lawyer, so you could check with Zion by creating a ticket internally.

Also, the workflow makes sense. Do I need to sign up with the email in my Github account? Does it matter?

Reply:
@jpsamper2009 it has to be the email which you use in your commits, since there is a check for that.

Have a look at this steps on how to proceed with the contribution https://github.com/eclipse/iceoryx/blob/master/CONTRIBUTING.md#feature-request-and-bugs
I think you can open the PR directly towards master and skip staging